src/realm-roles/AssociatedRolesTab.tsx

```diff
@@ -48,25 +59,39 @@ export const AssociatedRolesTab = ({
     </>
   );
 
-  const boolFormatter = (): IFormatter => (data?: IFormatterValueType) => {
-    const boolVal = data?.toString();
-
-    return (boolVal ? toUpperCase(boolVal) : undefined) as string;
-  };
+  const toggleModal = () => setOpen(!open);
 
   const [toggleDeleteDialog, DeleteConfirm] = useConfirmDialog({
     titleKey: "roles:roleRemoveAssociatedRoleConfirm",
-    messageKey: t("roles:roleRemoveAssociatedText", {
-      selectedRoleName: selectedRole ? selectedRole!.name : "",
-    }),
+    messageKey: t("roles:roleRemoveAssociatedText"),
     continueButtonLabel: "common:delete",
     continueButtonVariant: ButtonVariant.danger,
     onConfirm: async () => {
       try {
-        // await adminClient.roles.delCompositeRoles({ id: compID }, compies);
+        await adminClient.roles.delCompositeRoles({ id }, selectedRows);
+        setSelectedRows([]);
 
-        setSelectedRole(undefined);
-        addAlert(t("roleDeletedSuccess"), AlertVariant.success);
+        addAlert(t("associatedRolesRemoved"), AlertVariant.success);
+      } catch (error) {
+        addAlert(`${t("roleDeleteError")} ${error}`, AlertVariant.danger);
```

Comment:
it's better to do this concatenation in the message key
```suggestion
        addAlert({t("roleDeleteError", {error}), AlertVariant.danger);
```
then in the message bundle use:
```json
"roleDeleteError": "Could not delete role: {{error}}"
```